pkg/alertmanager/amcfg_test.go

```diff
@@ -4242,3 +4244,13 @@ func parseURL(t *testing.T, u string) *config.URL {
 	require.NoError(t, err)
 	return &config.URL{URL: url}
 }
+
+func newNopLogger(t *testing.T) *slog.Logger {
+	t.Helper()
+	return slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{
+		// slog level math.MaxInt means no logging
+		// We would like to use the slog buil-in No-op level once it is available
+		// More: https://github.com/golang/go/issues/62005
+		Level: slog.Level(math.MaxInt),
+	}))
+}
```

Comment:
Is it makes sense to create a helper function instead of duplicate this code across all the tests?

Reply:
Yes, for sure!